mpyl-example.py

```diff
@@ -78,6 +78,7 @@ def main(log: Logger, args: argparse.Namespace):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Simple MPL pipeline')
     parser.add_argument('--local', '-l', help='a local developer run', default=False, action='store_true')
+    parser.add_argument('--tag', '-t', help='A tag run to deploy to test/acce/prod', default=False, action='store_true')
```

Comment:
I think we probably need a target (test/acc/prod) parameter too. Not for this PR, but we will need it I think.

Reply:
It's handled by an environment variable in the run properties. Check also https://github.com/Vandebron/Vandebron/pull/10482